components/movementsensor/rtkutils/gpsrtk_test.go

```diff
@@ -1,99 +1 @@
 package rtkutils
-
```

Comment:
Mild preference to just delete the entire file. Its presence implies that tests exist.